homeassistant/components/climate/const.py

```diff
@@ -1,19 +1,75 @@
 """Provides the constants needed for component."""
 
+# All activity disabled / Device is off/standby
+OPERATION_MODE_OFF = 'off'
+
+# Heating
+OPERATION_MODE_HEAT = 'heat'
+
+# Cooling
+OPERATION_MODE_COOL = 'cool'
+
+# The device supports heating/cooling to a range
+OPERATION_MODE_HEATCOOL = 'heatcool'
+
+# The temperature is set based on a schedule, learned behavior, AI or some
+# other related mechanism. User is not able to adjust the temperature
+OPERATION_MODE_AUTO = 'auto'
+
+# Device is in Dry/Huminity mode
+OPERATION_MODE_DRY = 'dry'
+
+# Only the fan is on, not fan and another mode like cool
+OPERATION_MODE_FAN_ONLY = 'fan_only'
+
+OPERATION_MODES = [
+    OPERATION_MODE_OFF,
+    OPERATION_MODE_HEAT,
+    OPERATION_MODE_COOL,
+    OPERATION_MODE_HEATCOOL,
+    OPERATION_MODE_AUTO,
+    OPERATION_MODE_DRY,
+    OPERATION_MODE_FAN_ONLY,
+]
+
+
+# Device is running an energy-saving mode
+HOLD_MODE_ECO = 'eco'
+
+# Device is on away mode
+HOLD_MODE_AWAY = 'away'
+
+# Device turn all valve full up
+HOLD_MODE_BOOST = 'boost'
+
+# Device have a comfort mode
+HOLD_MODE_COMFORT = 'comfort'
+
+
+# This are support current states of HVAC
+OPERATION_CURRENT_OFF = 'off'
+OPERATION_CURRENT_HEAT = 'heat'
```

Comment:
We should make these values present participle: off, heating, cooling, drying, idle, fan_only